BD-2973 Hidden page for Dashboard Builder EA (#7160)
braze-inc/braze-docs: _docs/_hidden/private_betas/dashboard_builder.md

---
nav_title: Dashboard Builder
article_title: Dashboard Builder
permalink: "/dashboard_builder/"

> description: "This reference article covers how to save a draft for a campaign that has already launched."

Reviewer: ```suggestion
description: "This reference article covers how to use Dashboard Builder to create dashboards and visualizations using reports created in Query Builder."
```
👀

Author: Oh my gosh 😮‍💨 Thank you for catching this

page_type: reference
hidden: true
---

# Dashboard Builder

> > Use Dashboard Builder to create dashboards and visualizations using reports created in Query Builder. You can get started with the pre-built SQL query templates in Query Builder, or write your own custom SQL queries to unlock even more insights. 

Reviewer: ```suggestion
> Use Dashboard Builder to create dashboards and visualizations using reports created in Query Builder. You can get started with the pre-built SQL query templates in Query Builder or write your own custom SQL queries to unlock even more insights.
```

Braze provides pre-built dashboard templates for frequent use cases such as analysis of revenue using last-touch attribution. Note that the ability to edit a template dashboard is not yet available.

{% alert note %}
Dashboard Builder is currently in early access. Contact your customer success manager if you're interested in participating in this early access.
{% endalert %}

## Running a dashboard template

> 1. Go to **Analytics** > **Dashboard Builder**. The home page lists all existing dashboards within your workspace, with Braze created templates at the top. These are denoted with "(Braze)" in the title.
> 2. Select the dashboard you’re interested in.
> 3. Select **Run Dashboard** to generate a dashboard using that template.

Reviewer: ```suggestion
1. Go to **Analytics** > **Dashboard Builder**. The home page lists all existing dashboards within your workspace, with Braze-created templates at the top. These are denoted with "(Braze)" in the title.
2. Select the dashboard you’re interested in.
3. Select **Run Dashboard** to generate a dashboard using that template.
```

### Available dashboard templates

#### Last Touch Attribution - Revenue

| Metrics | Definition |
| --- | --- |

> | Total Last Touch Revenue | A sum of all campaign and Canvas revenue events that had a last-touch event within the selected date range and attribution window. |
> | Total Purchase Conversions | A count of all campaign and Canvas revenue events that had a qualifying last-touch event. |
> | Average Days to Convert | The average time between all campaign and Canvas purchase events that had a qualifying last-touch event. |
> | Revenue per Recipient | Sum of the revenue from qualified revenue events divided by the number of unique users that received a message within the date range. |
> | Unique Purchasers | Count of the unique users with a qualified revenue event. |
> | Revenue by Country | Sum of all campaign and Canvas revenue events that had a last-touch event, grouped by country. |
> | Revenue by Campaign | Sum of all campaign and Canvas revenue events that had a qualifying last-touch event, grouped by campaign. |
> | Revenue by Canvas | Sum of all campaign and Canvas revenue events that had a qualifying last-touch event, grouped by Canvas. |
> | Revenue by Channel | Sum of all campaign and Canvas revenue events that had a qualifying last-touch event, grouped by Canvas. | 
> | Revenue Time Series | Sum of all campaign and Canvas revenue events that had a qualifying last-touch event, grouped by day in UTC. |

Reviewer: ```suggestion
| Total Last Touch Revenue | A sum of all campaign and Canvas revenue events with a last-touch event within the selected date range and attribution window. |
| Total Purchase Conversions | A count of all campaign and Canvas revenue events with a qualifying last-touch event. |
| Average Days to Convert | The average time between all campaign and Canvas purchase events with a qualifying last-touch event. |
| Revenue per Recipient | Sum of the revenue from qualified revenue events divided by the number of unique users that received a message within the date range. |
| Unique Purchasers | Count of the unique users with a qualified revenue event. |
| Revenue by Country | Sum of all campaign and Canvas revenue events with a last-touch event, grouped by country. |
| Revenue by Campaign | Sum of all campaign and Canvas revenue events with a qualifying last-touch event, grouped by campaign. |
| Revenue by Canvas | Sum of all campaign and Canvas revenue events with a qualifying last-touch event, grouped by Canvas. |
| Revenue by Channel | Sum of all campaign and Canvas revenue events with a qualifying last-touch event, grouped by Canvas. | 
| Revenue Time Series | Sum of all campaign and Canvas revenue events with a qualifying last-touch event, grouped by day in UTC. |
```
Suggestion to replace "that had" with "with" - feel free to disregard!

{: .reset-td-br-1 .reset-td-br-2}

#### Devices and carriers

| Metrics | Definition |
| --- | --- |
| Device Carriers | Count of users in the selected date range who opened a push notification, grouped by device carrier. |

> | Device Model | Count of users in the selected date range who opened a push notificatio, grouped by device model. |

Reviewer: ```suggestion
| Device Model | Count of users in the selected date range who opened a push notification, grouped by device model. |
```

| Device Operating System | Count of users in the selected date range who opened a push notification, grouped by device operating system. |
| Device Screen Size | Count of users in the selected date range who opened a push notification, grouped by device screen resolution (size). |
{: .reset-td-br-1 .reset-td-br-2}

## Creating a customized dashboard

> 1. Select **Create Dashboard**, or an existing dashboard and **Edit**. Then select **+ Add Tile**.
> 2. Select **Select Existing Query** and choose a query that you've run in Query Builder.
> 3. To edit how the query results display in the tile, select the pencil icon to change the title and chart type.<br><br>![Tile editor view with options to change the title and chart type.][2]{: style="max-width:60%;"}<br><br>
>     - If selecting a chart type of **Column**, **Bar**, or **Line**:
>         - Select a field from the query to use for your X-axis.
>         - De-select the columns you're not interested in.<br><br>![Dropdown with the chart types.][1]{: style="max-width:40%;"}
> 
> {: start="4"}        
> 4. Make sure to save your changes. If you want to delete the tile, select the trash can icon. Note that deleted tiles can't be reverted and must be recreated.
> 5. Adjust the tile size by dragging the bottom right corner, and the tile position on the dashboard by dragging the handle at the top right corner.<br><br>![Tile getting dragging by the handle.][3]<br><br>
> 6. Add additonal tiles until your dashboard is complete.
> 7. Select **View Dashboard**, and then select **Run Dashboard**. Your dashboard might take up to a few minutes to finish generating reports.

Reviewer: ```suggestion
1. Select **Create Dashboard**, or an existing dashboard and **Edit**. Then select **+ Add Tile**.
2. Select **Select Existing Query** and choose a query you've run in Query Builder.
3. To edit how the query results display in the tile, select the pencil icon to change the title and chart type.<br><br>![Tile editor view with options to change the title and chart type.][2]{: style="max-width:60%;"}<br><br>
    - If selecting a chart type of **Column**, **Bar**, or **Line**:
        - Select a field from the query to use for your X-axis.
        - De-select the columns you're not interested in.<br><br>![Dropdown with the chart types.][1]{: style="max-width:40%;"}

{: start="4"}        
4. Make sure to save your changes. If you want to delete the tile, select the trash can icon. Deleted tiles can't be reverted and must be recreated.
5. Adjust the tile size by dragging the bottom right corner, and the tile position on the dashboard by dragging the handle at the top right corner.<br><br>![Tile getting dragging by the handle.][3]<br><br>
6. Add additional tiles until your dashboard is complete.
7. Select **View Dashboard**, and then select **Run Dashboard**. Your dashboard might take up to a few minutes to finish generating reports.
```
